### PR TITLE
added newPost page and started using components

### PR DESCRIPTION
**Pull request by Aamr-mohamed**

### CODE REVIEW

File: src/routes/Signup/signup.js
Comment (se-omarayman) [inline src/routes/Signup/signup.js:1]: sus naming

**Review by se-omar — CHANGES_REQUESTED** [review]
1. posts should have a title and a description.
2. posts should be linked to the user that created it, currently if im not logged in the user is saved as an empty string.
3. pics paths aren't saved in localstorage (check the saved path, it's C:\\fakepath\\) this is because a browser can't access the computer's disk, so adding images will be hard unless we have a real backend server to save the images to
4. add an indication to the user that the post was created (maybe alert)
5. post titles should be unique, so show an error if the post already exists in localstorage
6.

File: src/routes/Signup/signup.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Button from "react-bootstrap/Button";
 import bcrypt from "bcryptjs-react";
 import Form from "react-bootstrap/Form";
 import { FloatingLabel } from "react-bootstrap";
+import Buttonn from "../components/button";
```
Comment (se-omar) [inline src/routes/Signup/signup.js]: sus naming

Comment (se-omar) [reply src/routes/Signup/signup.js]: also the file name for the component should match the component name

**Comment by se-omar** [comment]
in posts:
1. we need to store the user email, not the username bec. email is the unique key.
2. we need to store the time of creating the post, call it createdAt.
3. remove the select file page, since we can't store files in local storage.
4. the property name should be posts, not addPost (in localstorage)

**Pull request merged**